packages/next/src/elements/DocumentHeader/Tabs/index.tsx

```diff
@@ -32,47 +32,100 @@ export const DocumentTabs: React.FC<{
 
   const customViews = getCustomViews({ collectionConfig, globalConfig })
 
+  const defaultTabs = tabs(collectionConfig, globalConfig)
```

Comment:
Let's rename this to something like `getTabs` and send a single argument which is a keyed object, similar to `getCustomViews` above.